tests/test_units/test_global_configuration.py

```diff
@@ -88,3 +88,63 @@ def test_access_classification_logic():
                 "ssdl": {"rep_include": False},
             }
         )
+
+
+def test_stratigraphy(edataobj2):
+    """Test the stratigraphy."""
+
+    strat = edataobj2.config.stratigraphy
+
+    ###### Test get index ######
+
+    # Correct name
+    assert strat._get_index("Valysar") == 9
+
+    # Correct name
+    assert strat._get_index("Therys") == 10
+
+    # Incorrect name
+    with pytest.raises(ValueError):
+        strat._get_index("Garn")
+
+    # Empty name
+    with pytest.raises(ValueError):
+        strat._get_index("")
+
+    ###### Test get official name ######
+
+    # Correct name
+    assert "Valysar Fm.".__eq__(strat.get_official_name("Valysar"))
+
+    # Incorrect name
+    assert "".__eq__(strat.get_official_name("Ile"))
+
+    # Empty name
+    assert "".__eq__(strat.get_official_name(""))
```

Comment:
Why `__eq__` over `==`?

Reply:
Sorry, did some last minute testing and forgot to correct.